src/thread.rs

```diff
@@ -96,7 +96,8 @@ mod test {
             let time_before = time::SteadyTime::now();
```

Comment:
There's no conflict - `std` does not have `SteadyTime`. So all we need is `use time::SteadyTime` and `SteadyTime::now()`. I think that would be fine. Same for the other comment here.